app/cdap/components/InlayDrawerWidget/index.tsx

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Button, IconButton, Menu, MenuItem, Typography } from '@material-ui/core';
+import grey from '@material-ui/core/colors/grey';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import CloseRoundedIcon from '@material-ui/icons/CloseRounded';
+import { PREFIX } from 'components/InlayDrawerWidget/InlayDrawerWidget.stories';
+import T from 'i18n-react';
+import React, { PropsWithChildren } from 'react';
+import styled from 'styled-components';
+
+export interface IMenuItem {
+  clickHandler: () => void;
+  label: string;
+  value: string;
+}
+
+interface IRecipeStepWidgetProps {
+  actionsOptions: IMenuItem[];
+  headingText: string;
+  onClose: () => void;
+  position: 'left' | 'right';
+  disableActionsButton?: boolean;
+  showDivider?: boolean;
+}
+
+const Container = styled(Box)`
+  border-left: 1px solid ${grey[300]};
+  height: calc(100vh - 232px);
+  overflow: scroll;
+  padding-left: 20px;
+  padding-right: 10px;
+  position: relative;
+  width: 500px;
+`;
+
+const Divider = styled.div`
```

Comment:
can you break out these styles into a different file to make this file shorter

Reply:
moved the styled-components to a separate file, named styles.ts